setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
The "standard" name of the section is to name them after the command name (e.g. you can have an ``[sdist]`` or an ``[upload]`` section or whatever). The old name was just something Daniel did and it got switched to be more inline with what the rest of the commands used.